src/client/components/UserForm.jsx

```diff
@@ -1,27 +1,42 @@
 import { useState } from "react";
 
-export default function UserForm({ handleSubmit }) {
-    const [user, setUser] = useState({ username: '', password: '' });
+export default function UserForm({ handleSubmit, title }) {
+  const [user, setUser] = useState({ username: "", password: "" });
 
-    const handleSubmitDecorator = (e) => {
-        e.preventDefault();
-        handleSubmit(user);
-    };
+  const handleSubmitDecorator = (e) => {
+    e.preventDefault();
+    handleSubmit(user);
+  };
 
-    const handleChange = (e) => {
-        const { name, value } = e.target;
+  const handleChange = (e) => {
+    const { name, value } = e.target;
 
-        setUser({
-            ...user,
-            [name]: value
-        });
-    };
+    setUser({
+      ...user,
+      [name]: value,
+    });
+  };
 
-    return (
-        <form onSubmit={handleSubmitDecorator}>
-            <input type="text" name="username" placeholder="Username" value={user.username} onChange={handleChange} />
-            <input type="password" name="password" placeholder="Password" value={user.password} onChange={handleChange} />
-            <button type="submit">Submit</button>
-        </form>
-    );
-}
\ No newline at end of file
+  return (
+    <>
+      <h1>{String(title)}</h1>
```

Comment:
What's the reasoning here to invoke String as a method?

Reply:
I have no idea why I did this! Removed the method call ✔️